Makefile

```diff
@@ -7,4 +7,4 @@ test-dev        :; ./test-dssspell.sh match="$(match)" optimizer="0"
 test-forge      :; ./test-dssspell-forge.sh match="$(match)" block="$(block)"
 deploy          :; make && dapp create DssSpell | xargs ./verify.py DssSpell
 flatten         :; hevm flatten --source-file "src/Goerli-DssSpell.sol" > out/flat.sol
-archive-spell   :; ./archive-dssspell.sh $(name)
+archive-spell   :; ./archive-dssspell.sh "$(if $(date),$(date),$(shell date +'%Y-%m-%d'))"
```

Comment:
This is really confusing as both the expected env var is called "date" and the shell cmd is called "date".
The general standard is to have env vars capitalized (which will requie changing also" match" and "block" above).
I don't mind if we leave it as is for now and change it later.

Reply:
Ah nvm, seeing now that since it's a makefile you pass date=X so I guess it doesn't have to be an env var.
So You can ignore it.